prime-router/src/main/kotlin/Element.kt

```diff
@@ -893,7 +945,11 @@ data class Element(
     companion object {
         const val datePattern = "yyyyMMdd"
         const val datetimePattern = "yyyyMMddHHmmZZZ"
-        const val variableDateTimePattern = "[yyyyMMddHHmmssZ][yyyyMMddHHmmZ][yyyyMMddHHmmss]"
+        // isn't she a beauty? This allows for all kinds of possible date time variations
+        const val variableDateTimePattern = "[yyyyMMddHHmmssZ]" +
+            "[yyyyMMddHHmmZ]" +
+            "[yyyyMMddHHmmss][yyyy-MM-dd HH:mm:ss.ZZZ]" +
+            "[yyyy-MM-dd[ HH:mm:ss[.S[S][S]]]]"
```

Comment:
This is the real magic of the date time parsing. We're using the variable date time pattern above and then letting Java do a `parseBest` based on what matches.